Automattic/jetpack
# [RNMobile] Fix - Only open upload options automatically when the block is inserted

## Proposed changes:

* The upload options were automatically opened every time the block was selected. Now, similarly to other media blocks, the upload options are only opened when the block was just inserted.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






1. Tap ➕ to open the block inserter.
2. Add VideoPress block.
3. Observe that the media upload options are opened automatically.
4. Observe that other actions that create the block (i.e. actions that don't imply inserting it from the block inserter) don't open automatically the media upload options.
    - Copy and paste the block.
    - Cut and paste the block.
    - Duplicate the block.
5. Having the VideoPress block in its empty state (showing the placeholder), unselect it by selecting a different block.
6. Tap on the VideoPress block again.
7. Observe that the block is selected and that the media upload options are opened automatically.
